src/material-experimental/mdc-button/button.scss

```diff
@@ -45,3 +45,15 @@
   right: -$mdc-button-outlined-border-width;
   border: none;
 }
+
+// MDC expects button icons to contain this HTML content:
+// ```html
+//   <span class="mdc-button__icon material-icons">favorite</span>
+// ```
+// However, Angular Material expects a `mat-icon` instead. The following
+// will extend the `mdc-button__icon` styling to the mat icon. Note that
+// the extended styles inherently only match icons that nest themselves in
+// a parent `mdc-button`.
+.mat-icon {
```

Comment:
Should this be constrained to only `mat-icon` inside of `mat-icon-button`?

Reply:
We seem to have an identical issue in the FAB. I've submitted https://github.com/angular/components/pull/20969 to address it.

Reply:
Yeah I may have copy/pasted this from fab 😅

Reply:
Still looks like this needs to be changed?

Reply:
Sorry, forgot to push